quarkus/service/src/main/resources/application.properties

```diff
@@ -120,30 +120,18 @@ polaris.authentication.token-broker.max-token-generation=PT1H
 # polaris.storage.gcp.token=token
 # polaris.storage.gcp.lifespan=PT1H
 
+# Profile for unit tests
 %test.quarkus.log.file.enable=false
-%test.quarkus.log.category."org.apache.polaris".level=INFO
-%test.quarkus.log.category."org.apache.iceberg.rest".level=INFO
-%test.quarkus.log.category."org.apache.iceberg.rest.RESTSessionCatalog".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.core.persistence.PolarisMetaStoreManagerImpl".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.service.context.DefaultRealmContextResolver".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.service.catalog.PolarisCatalogHandlerWrapper".level=ERROR
-%test.quarkus.log.category."org.apache.polaris.service.storage.PolarisStorageIntegrationProviderImpl".level=ERROR
-%test.quarkus.http.limits.max-body-size=1000000
 %test.quarkus.otel.sdk.disabled=true
-%test.polaris.authentication.token-broker.type=symmetric-key
-%test.polaris.authentication.token-broker.symmetric-key.secret=polaris
-%test.polaris.features.defaults."ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING"=false
-%test.polaris.features.defaults."ALLOW_EXTERNAL_METADATA_FILE_LOCATION"=false
-%test.polaris.features.defaults."ALLOW_OVERLAPPING_CATALOG_URLS"=true
-%test.polaris.features.defaults."ALLOW_SPECIFYING_FILE_IO_IMPL"=true
-%test.polaris.features.defaults."ALLOW_WILDCARD_LOCATION"=true
-%test.polaris.features.defaults."ENFORCE_PRINCIPAL_CREDENTIAL_ROTATION_REQUIRED_CHECKING"=true
-%test.polaris.features.defaults."INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST"=true
-%test.polaris.features.defaults."SKIP_CREDENTIAL_SUBSCOPING_INDIRECTION"=true
-%test.polaris.features.defaults."SUPPORTED_CATALOG_STORAGE_TYPES"=["FILE","S3","GCS","AZURE"]
-%test.polaris.realm-context.realms=POLARIS
-%test.polaris.realm-context.type=test
-%test.polaris.storage.aws.access-key=accessKey
-%test.polaris.storage.aws.secret-key=secretKey
-%test.polaris.storage.gcp.token=token
-%test.polaris.storage.gcp.lifespan=PT1H
+
+# Profile for integration tests
+%it.quarkus.http.limits.max-body-size=1000000
```

Comment:
Unfortunately, when running integration tests, the configuration needs to be built into the application jar. But at least, the changes are confined in a specific profile that will be used only for that.

Reply:
This change LGTM, but it also highlights my concerns with running body size restrictions as part of our "normative" suite of integration tests (commented under #630). It is very specific to the server env. FWTW, even HTTP proxies may have their own (different) limits.